canonical-ols/build.snapcraft.io
# Landing page tweaks

## DONE

* Added testimonials to the landing page
* Added left, center, right align property on Heading module - sorry

## QA

1. go to the landing page
2. look at the testimonials... see that the look like the [design spec](https://www.dropbox.com/s/6je201c3jp2as2z/1%20-%20testimonials%20-%20no%20avatar.jpg?dl=0) and [trello card instructions](https://trello.com/c/bPbPoJ1u)
3. read them and see that they conform to the [copy doc](https://docs.google.com/document/d/1ijyI7EMPu0BoyVbB-mPB7gyKnPyloC0Rvx1Oq-yEqKk/edit#)
